CMakeLists.txt

```diff
@@ -821,14 +821,23 @@ message(STATUS "  Assertions: ${LLVM_ENABLE_ASSERTIONS}")
 message(STATUS "  LTO:        ${SWIFT_TOOLS_ENABLE_LTO}")
 message(STATUS "")
 
-message(STATUS "Building Swift standard library and SDK overlays for SDKs: ${SWIFT_SDKS}")
-message(STATUS "  Build type: ${SWIFT_STDLIB_BUILD_TYPE}")
-message(STATUS "  Assertions: ${SWIFT_STDLIB_ASSERTIONS}")
-message(STATUS "")
+if (SWIFT_BULID_STDLIB OR SWIFT_BUILD_SDK_OVERLAY)
 
-message(STATUS "Building Swift runtime with:")
-message(STATUS "  Leak Detection Checker Entrypoints: ${SWIFT_RUNTIME_ENABLE_LEAK_CHECKER}")
-message(STATUS "")
+  message(STATUS "Building Swift standard library and SDK overlays for SDKs: ${SWIFT_SDKS}")
+  message(STATUS "  Build type: ${SWIFT_STDLIB_BUILD_TYPE}")
+  message(STATUS "  Assertions: ${SWIFT_STDLIB_ASSERTIONS}")
+  message(STATUS "")
+
+  message(STATUS "Building Swift runtime with:")
+  message(STATUS "  Leak Detection Checker Entrypoints: ${SWIFT_RUNTIME_ENABLE_LEAK_CHECKER}")
+  message(STATUS "")
+
+else()
+
+  message(STATUS "Not building Swift standard library and runtime")
```

Comment:
Not building Swift standard library, *SDK overlays*, or runtime, right?

Reply:
I'd thought about saying that when I was first writing it out, but lazed out at the last minute, thinking it would be a "duh". Apparently not.

Also, I was wondering if you folks preferred an Oxford comma or not. Luckily, your comment itself made it easy for me to figure that one out :)

Reply:
It wasn't a purposeful Oxford comma; I'm not sure that there's an answer to that :) Core team will weigh in here if they feel strongly.